src/Analysis/Engine/Impl/DependencyResolution/Arc.cs

```diff
@@ -0,0 +1,92 @@
+﻿// Python Tools for Visual Studio
+// Copyright(c) Microsoft Corporation
+// All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the License); you may not use
+// this file except in compliance with the License. You may obtain a copy of the
+// License at http://www.apache.org/licenses/LICENSE-2.0
+//
+// THIS CODE IS PROVIDED ON AN  *AS IS* BASIS, WITHOUT WARRANTIES OR CONDITIONS
+// OF ANY KIND, EITHER EXPRESS OR IMPLIED, INCLUDING WITHOUT LIMITATION ANY
+// IMPLIED WARRANTIES OR CONDITIONS OF TITLE, FITNESS FOR A PARTICULAR PURPOSE,
+// MERCHANTABILITY OR NON-INFRINGEMENT.
+//
+// See the Apache Version 2.0 License for specific language governing
+// permissions and limitations under the License.
+
+using System.Collections.Generic;
+
+namespace Microsoft.PythonTools.Analysis.DependencyResolution {
+    internal partial struct PathResolverSnapshot {
+        /// <summary>
+        /// Represents arc between two nodes in the tree
```

Comment:
I think it is called `edge` in https://en.wikipedia.org/wiki/Glossary_of_graph_theory_terms#edge

Reply:
![image](https://user-images.githubusercontent.com/8933858/47808785-27fcea80-dd0d-11e8-8df7-80e96c3755da.png)
but I can rename it if it is easier to understand.

Reply:
Also in the same vein of naming changes would be vertexes -> vertices, but I think "arc" is much less common relative to "edge" than "vertexes" is to "vertices".

Reply:
I am fine either way, just that more commonly used term is easier to understand for newcomers.

Reply:
Renamed